app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
```

Comment:
Code Style Issue: The variable names in the __init__ method of the Car class are descriptive, but they could be more specific to reflect their purpose and type. For example, 'comfort_class' could be renamed to 'comfort_class_level', 'clean_mark' could be renamed to 'clean_mark_score'.